### PR TITLE
Market 135 integration

### PR DESCRIPTION
**Pull request by ivy-lmu**

### CODE REVIEW

**Comment by ivy-lmu** [comment]
@andreasbalsiger we implemented another connector. your feedback is appreciated, thank you.

File: market/connector/tel-search-connector/meta.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+	"id": "tel-search-ch-connector",
+	"name": "tel.search.ch",
+	"description": "It has never been easier to retrieve phone directory entries via the tel.search.ch API.",
```
Comment (andreasbalsiger) [inline market/connector/tel-search-connector/meta.json:4]: Retrieve phone directory entries from Switzerland within your process work using the tel.search.ch API.

File: market/connector/tel-search-connector/meta.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
+{
+	"id": "tel-search-ch-connector",
+	"name": "tel.search.ch",
```
Comment (andreasbalsiger) [inline market/connector/tel-search-connector/meta.json:3]: Swiss phone directory

**Review by andreasbalsiger — COMMENTED** [review]
have a look at my ideas :D

**Review by alexsuter — APPROVED** [review]

**Comment by alexsuter** [comment]
ready2merge :)

**Pull request merged**